firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -78,7 +78,7 @@ public static FirebaseInstallations getInstance(@NonNull FirebaseApp app) {
   @NonNull
   @Override
   public Task<InstallationTokenResult> getAuthToken(boolean forceRefresh) {
-    return Tasks.forResult(InstallationTokenResult.create("dummy_auth_token", 1000l));
+    return Tasks.forResult(InstallationTokenResult.builder().build());
```

Comment:
Can you make the two members nonnull(token and expiration) in InstallationTokenResult? So it'll not be okay to build a InstallationTokenResult with token or expiration

Reply:
Added the nonull annotation, but that doesnt block the default instance creation. Leaving it as is for now.